packages/runtime/test/transport/relationships.test.ts

```diff
@@ -608,7 +667,18 @@ describe("RelationshipTermination", () => {
                 to: [services2.address]
             }
         });
-        expect(result).toBeAnError(/.*/, "error.transport.messages.missingOrInactiveRelationship");
+        expect(result).toBeAnError(/.*/, "error.transport.messages.hasNoActiveRelationship");
+    });
+
+    test("should be able to send a Notification", async () => {
```

Comment:
all this notification stuff should be in the test file for notifications

Reply:
But here the focus is on the RelationshipTermination and we also test, that a Message is not possible to send in this case.

Reply:
you are testing that the SendMessage UseCase reacts correctly on the different Relationship statuses.

But WHAT you test is the SendMessage UseCase and this decides where to put the test.

Reply:
Ehm, I think I messed up. In the first comment I wrote, that these tests belong to the notifications file, but as this tests the sendMessage UseCase it defenitely should be part of the messages tests. Sorry for the confusion.

Reply:
So also the test 'should not send a message' should be in the messages test, isn't so?